config/cypress/support.js

```diff
@@ -11,6 +11,20 @@ if (Cypress.env("CYPRESS_COVERAGE")) {
 // NOTE: If we end up with a lot of custom commands, we should break
 // each command into its own file.
 
+// NOTE(Jeremy): Leaving this here for the future where we migrate this to TS.
+// It works, but switching our Cypress config to .ts files causes Cypress types
+// to conflict with Jest types. The following URL looks like it would fix it,
+// but I couldn't get it working in a short timebox. Leaving this breadcrumb
+// trail for future!
+// https://docs.cypress.io/guides/tooling/typescript-support#Clashing-Types-with-Jest
+// declare global {
+//     namespace Cypress {
+//         interface Chainable {
+//             dragTo(position: {x: number; y: number}): Chainable<any>;
+//         }
+//     }
+// }
```

Comment:
Should this have a TODO linked to a Jira issue? Otherwise, I suspect we'll probably forget to come back to this.

Reply:
Good call.  LC-1495 now exists to cover migrating our Cypress configs to Typescript.